Makefile

```diff
@@ -266,28 +266,34 @@ set_govulncheck:
 govulncheck: set_govulncheck tidy-vendor
 	@govulncheck -v ./... || true
 	
-cluster-clean: build-manifest
-	./hack/cluster-clean.sh
 .PHONY: cluster-clean
+cluster-clean:
+	./hack/cluster-clean.sh
 
+.PHONY: cluster-deploy
 cluster-deploy: cluster-clean
 	BARE_METAL_NODE_ONLY=false ./hack/cluster-deploy.sh
-.PHONY: cluster-deploy
 
+.PHONY: cluster-sync
 cluster-sync:
 	./hack/cluster-sync.sh
-.PHONY: cluster-sync
 
+.PHONY: prepareKubeConfig
 prepareKubeConfig:
 	./hack/prepareKubeConfig.sh
-.PHONY: prepareKubeConfig
 
-cluster-up:
-	rm -rf local-dev-cluster
-	git clone -b v0.0.1 https://github.com/sustainable-computing-io/local-dev-cluster.git --depth=1
-	cd local-dev-cluster && ./main.sh
 .PHONY: cluster-up
+cluster-up:
```

Comment:
Can we move the code to a separate script inside the hack folder? Like we have in [Kepler](https://github.com/sustainable-computing-io/kepler/blob/main/hack/cluster-up.sh) repo

Reply:
yes, but beyond the scope of this PR :)

Reply:
@vprashar2929 please feel free to open PR since I think there is value in keeping all non-trivial scripts in `/hack`